ral-facilities/operationsgateway
# DSEGOG-67 additional plot styling options

## Description
This PR adds:
- ability to toggle plot gridlines (for both axes)
- ability to toggle axes labels (currently only displaying a label for x-axis, since we haven't finalised the y-axes and their respective labelling yet)
- ability to add minimum and maximum limits for both x- and y-axes
- ability to change line style between solid, dashed and dotted

The amount of options per data channel means there's now a lot of buttons clogging up the label space. I'm open to adding some sort of popup menu per data channel - the user clicks a button, which opens a popup showing the options for each channel, which they can edit there

## Testing instructions
Try out the above features mixed with each other to make sure they behave independently


## Agile board tracking
Closes DSEGOG-67
